certification/internal/policy/container/base_on_ubi_test.go

```diff
@@ -81,7 +98,22 @@ NAME="Red Hat Enterprise Linux"
 		os.RemoveAll(imageRef.ImageFSPath)
```

Comment:
You can also remove all this junk that was for the previous way of doing this check.

Reply:
this becomes redundant yeah but i guess CVP does both ?

Reply:
I was unaware of that. If that's the case, I guess we should leave it in, but... A better test would be to check only the top layer? Perhaps we add that back at a later date?

Reply:
yeah because this might break at some point as it is different between ubi-7 and ubi-8 .. so removed it ..